packages/Liquid-Core.package/LiquidHostMenu.class/instance/closePoll.st

```diff
@@ -1,6 +1,8 @@
 change behavior
 closePoll
+	| pollId |
+	pollId := UIManager default request: 'Enter Poll ID'.
 	(PollRepo default
-		at: (UIManager default request: 'Enter Poll ID')
+		at: pollId
 		ifAbsent: [^ UIManager default inform: 'This poll does not exist.'])
-		isOpen: false
\ No newline at end of file
+		closeWithPassword: (LQPasswordManager default findPasswordFor: pollId)
```

Comment:
[Boulder] When attempting to close a Poll you don't have the Password for, you get an Error "Key not found". We need to catch that error and give UI-feedback to the User informing them that they don't have permission to access that poll.